yarn.lock

```diff
@@ -117,6 +117,13 @@
   resolved "https://registry.yarnpkg.com/@colors/colors/-/colors-1.5.0.tgz#bb504579c1cae923e6576a4f5da43d25f97bdbd9"
   integrity sha512-ooWCrlZP11i8GImSjTHYHLkvFDP48nS4+204nGb1RiX/WXYHmJA2III9/e2DWVabCESdW7hBAEzHRqUn9OUVvQ==
 
+"@cspotcode/source-map-support@^0.8.0":
```

Comment:
Not sure if it's included already or not, but would you mind also addressing the two dependabot PRs with this as well? Or just merge them after this one maybe, whichever is easier.

Reply:
Merged in the two updates.